Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -190,16 +207,52 @@ public class CarPlayMapViewController: UIViewController {
         styleManager?.styles = styles
     }
     
+    func setupSpeedLimitView() {
+        let speedLimitView = SpeedLimitView()
+        speedLimitView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(speedLimitView)
+        
+        speedLimitView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 8).isActive = true
+        speedLimitView.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -8).isActive = true
```

Comment:
nit: Don't we support RTL languages? It is recommended to use trailing/leading instead of right/left. Although it requires all other views to use t/l as well. Probably we need an epic for RTL support.

Reply:
Not sure if RTL support is necessary here, afaik this view doesn't contain any localized text.
I would rather think about supporting right-wheel cars, we need to make sure UI looks good there as well.

Reply:
It's hard to check behavior with right-to-left languages. Even though it can be changed via project scheme CarPlay will not be affected and trip preview view will stay unchanged. Once we have real hardware I'll retest this.